homeassistant/components/zwave_js/services.yaml

```diff
@@ -29,3 +29,37 @@ set_lock_usercode:
       example: 1234
       selector:
         text:
+
+set_config_parameter:
+  name: Set a Z-Wave device configuration parameter.
+  description: Allow for changing configuration parameters of your Z-Wave devices.
+  target:
+  fields:
+    entity_id:
+      name: Entity ID
+      description: One of the entity_id's of the device you want to configure.
+      example: light.zwave_dimmer
+    device_id:
+      name: Device ID
+      description: The device ID of the device you want to configure.
+      example: 336bef715bf7d370934564d4d8ef8457
```

Comment:
As per comment here: <https://github.com/home-assistant/core/pull/46673#discussion_r580382233>


This will show selectable entities (and area's/devices that those entities are in) based on the fact if those entities are provided by the Z-Wave JS integration.

```suggestion
set_config_parameter:
  name: Set a Z-Wave device configuration parameter
  description: Allow for changing configuration parameters of your Z-Wave devices.
  target:
  	entity:
      integration: zwave_js
  fields:
```

Tested against the dev frontend for the 2021.3 release

![image](https://user-images.githubusercontent.com/195327/108738481-ede63100-7533-11eb-8539-64686a295c49.png)

Reply:
Nice!